### DISCUSSION TITLE
【求助】传奇绳匠法厄同需要你的帮助

### DISCUSSION THREAD

**Discussion by lzxixizi** [share121/inter-knot#1208] [其他]
我是传奇绳匠法厄同，50年前天外势力入侵地球，向地球投放了大量的空洞灾害。我法厄同死守太空防线，怎知敌人以骸阴险狡诈，趁我突破境界之时偷袭我，使我在此役中身陨。如今我已重生归来，现急需网友的协助，请各位v我50购买鸣徽，待我战胜尼尼微，清除空洞灾害，封你做对空7部部长。

**Comment by llxcx4** [comment]
我是厨师，这就是甜菜↑

**Comment by Sumizawa** [comment]
你信我是秦始皇还是信他

**Comment by gaojiaobei** [comment]
法厄同已经渗透进官方了吗，不愧是传奇绳匠啊

**Comment by ZhuYuan-148** [comment]
我可不记得有这么多法厄同…编号148，你被逮捕了
![Screenshot_20240810_214721](https://github.com/user-attachments/assets/2585510a-a9bf-4821-a39a-430dae4cff4a)

**Comment by mx6436** [comment]
这是假的，法厄同发过视频提醒大家谨访诈骗[https://www.bilibili.com/video/BV1GJ411x7h7](url)
